### PR TITLE
[Connect] Prepare to publish StripeConnect module

### PR DESCRIPTION
**Pull request by mludowise-stripe**
## Summary
Prepare for StripeConnect SDK release:
- Adds `StripeConnect` to package managers and modules.yaml
- Configures installation tests
- Adds to modules list in main README
- Adds changelog entry

## Motivation
https://jira.corp.stripe.com/browse/MXMOBILE-2496

## Testing
- [x] Installation tests pass on CI
- [x] Verify size report running on Emerge Tools:
   https://www.emergetools.com/build/14436e05-f195-4a1c-9b30-fc1950750883

### CODE REVIEW

File: modules.yaml [hunk 1]
```diff
@@ -189,6 +189,16 @@ modules:
...
 
+- podspec: StripeConnect.podspec
+  scheme: StripeConnect
+  framework_name: StripeConnect
+  localization_dir: StripeConnect/StripeConnect
+  custom_string_convertible_dir: StripeConnect/StripeConnect/Source
+  supports_catalyst: false
+  docs:
+    output: docs/stripeconnect
+    readme: StripeConnect/README.md
```
[the remark this answers is not in the document]
Comment (mludowise-stripe) [reply modules.yaml:197]: https://github.com/stripe/stripe-ios/pull/4330

File: StripeConnect.podspec [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  s.summary                        = 'Use Connect embedded components to add connected account dashboard functionality to your app.'
+  s.license                        = { type: 'MIT', file: 'LICENSE' }
+  s.homepage                       = 'https://docs.stripe.com/connect/get-started-connect-embedded-components'
+  s.authors                        = { 'Stripe' => 'support+github@stripe.com' }
+  s.source                         = { git: 'https://github.com/stripe/stripe-ios.git', tag: "#{s.version}" }
+  s.frameworks                     = 'Foundation', 'WebKit', 'UIKit'
+  s.requires_arc                   = true
+  s.platform                       = :ios
+  s.swift_version = '5.0'
+  s.ios.deployment_target          = '13.0'
```
[the remark this answers is not in the document]
Comment (nschris-stripe) [reply StripeConnect.podspec:17]: Should we add this as a comment?

**Review by nschris-stripe — DISMISSED** [review]

**Comment by mludowise-stripe** [comment]
> # 📸 Snapshot Test
> ## Base build not found
> No build was found for the base commit [59e53e7](https://www.github.com/stripe/stripe-ios/commit/59e53e72f03c851409df39d2796b5e569afc63fb?utm_source=github&utm_medium=vcs). This is required to generate a snapshot diff for your pull request.
> 
> It's possible that you created a branch off the base commit before all of the CI steps have finished processing, e.g. the one that uploads a build to our system. If that's the case, no problem! Just wait and this will eventually resolve.
> 
> 🛸 Powered by [Emerge Tools](https://www.emergetools.com/?utm_source=github&utm_medium=vcs)

This is an expected failure because there were no previous builds of StripeConnect for Emerge to compare the size against

**Review by nschris-stripe — APPROVED** [review]

File: Tests/installation_tests/cocoapods/with_frameworks_swift/CocoapodsTest/Info.plist [hunk 3]
```diff
@@ -22,6 +22,8 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSCameraUsageDescription</key>
+	<string>camera description</string>
```
Comment (mludowise-stripe) [inline Tests/installation_tests/cocoapods/with_frameworks_swift/CocoapodsTest/Info.plist:26]: Connect assertion fails if the app doesn't configure this, so this has been added to all installation test projects

**Review by nschris-stripe — APPROVED** [review]

**Pull request merged**